### ISSUE TITLE
Auto-ranking of most explicative features

### ISSUE THREAD

**Issue by danthe3rd** [facebookresearch/hiplot#228]
**Scenario:**
I have a grid-search on parameters A, B and C.
For each sample, I have an associated `loss` which I try to minimize.

I want to know which parameter (A, B or C) has the most influence on the loss automatically.

**In python**: This can be done by learning a simple `RandomForestRegressor` (or Classifier depending on the target value type), and then calling `permutation_importance` to get an importance score for each parameter.
For this to be embedded in HiPlot, it would need to be done in JS (for example with [this library?](https://github.com/zemlyansky/random-forest))

https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html
https://scikit-learn.org/stable/modules/permutation_importance.html
